City-of-Helsinki/helsinki-paatokset
# Pp 297 voting results

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-297

**How to test**

Checkout branch and run `drush cr` and find a decision with votes i.ex. https://helsinki-paatokset.docker.so/fi/asia/hel-2021-002856/3e4074e1-8b43-4abf-a61a-d4d672b7c5c3

There should be a new accordion that contains the vote results and presents them first by amount of different votes and then by political group. Check that the data seems correct and check the code as well. Check that the accordion isn't visible is there aren't any votes.

Could the php be shortened somehow? Now there's a lot of the same rows for the different votes.
